### PR TITLE
updated pom

### PR DESCRIPTION
**Pull request by kdobija1**
JIRA Tickets: https://companieshouse.atlassian.net/browse/BI-11031 

removed Kafka clients dependency and removed log4j logging. switched to version of ch-kafka used by example project

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -22,13 +22,12 @@
         <rest-service-common-library.version>0.0.5</rest-service-common-library.version>
         <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
         <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
-        <ch-kafka.version>1.4.5</ch-kafka.version>
-        <structured-logging.version>1.9.11</structured-logging.version>
+        <ch-kafka.version>1.4.2</ch-kafka.version>
```
Comment (mreis-ch) [inline pom.xml]: Is this what implements kafka3? Seems strange to go back a few versions

Comment (kdobija1) [reply pom.xml]: It does go back slightly but it's the version of ch-kafka they used in https://github.com/companieshouse/insolvency-delta-consumer/blob/main/pom.xml which is our Kafka3 example

Comment (mreis-ch) [reply pom.xml]: I don't think we need to go backwards with the version of `ch-kafka` as this is our own library. The first bit in the ticket is to make sure the version of `sping-kafka` supports `kafka3` and if you look at https://spring.io/projects/spring-kafka#overview that version is 2.8.0 and is already set in the pom at line 30 so I think its good to go. Might be worth checking with Jo though

Comment (kdobija1) [reply pom.xml]: checking some of the other projects that use Kafka some use higher versions of ch-kafka than 1.4.2 so it should be fine to put it back to 1.4.5

File: pom.xml [hunk 2]
```diff
@@ -23,12 +23,11 @@
         <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
         <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
         <ch-kafka.version>1.4.5</ch-kafka.version>
-        <structured-logging.version>1.9.11</structured-logging.version>
+        <structured-logging.version>1.9.12</structured-logging.version>
         <junit-jupiter.version>5.5.2</junit-jupiter.version>
         <mockito.version>3.1.0</mockito.version>
         <kafka-models.version>1.0.19</kafka-models.version>
         <spring-kafka.version>2.8.0</spring-kafka.version>
```
Comment (mreis-ch) [inline pom.xml]: The latest version is 2.8.5, thinking maybe we should update to that version as we're updating the POM

File: pom.xml [hunk 3]
```diff
@@ -23,12 +23,11 @@
         <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
         <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
         <ch-kafka.version>1.4.5</ch-kafka.version>
-        <structured-logging.version>1.9.11</structured-logging.version>
+        <structured-logging.version>1.9.12</structured-logging.version>
```
Comment (mreis-ch) [inline pom.xml]: The latest version of structured logging is 1.9.14. Might as well update to that

**Review by mreis-ch — APPROVED** [review]

**Pull request merged**